bots/tests-scan

```diff
@@ -21,68 +21,74 @@
-    ],
-    'cockpit-project/cockpit-podman': [
-        'cockpit/fedora-29',
-    ],
-    'weldr/welder-web': [
-        'cockpit/fedora-29/chrome',
-        'cockpit/fedora-29/firefox',
-    ],
+REPO_BRANCH_CONTEXT = {
+    'cockpit-project/cockpit': {
+        'master': ['avocado/fedora', 'container/kubernetes', 'container/bastion',
+            'selenium/firefox', 'selenium/chrome', 'verify/centos-7', 'verify/debian-stable',
+            'verify/debian-testing', 'verify/fedora-29', 'verify/fedora-30', 'verify/fedora-atomic',
+            'verify/ubuntu-1804', 'verify/ubuntu-stable', 'verify/rhel-7-6-distropkg',
+            'verify/rhel-7-7', 'verify/rhel-8-0', 'verify/rhel-atomic', 'selenium/edge',
+            'verify/rhel-8-1',
+        ],
+        'rhel-7.6': ['avocado/fedora', 'container/kubernetes', 'container/bastion',
+            'selenium/firefox', 'selenium/chrome', 'verify/rhel-7-6',
+        ],
+        'rhel-7.7': ['avocado/fedora', 'container/kubernetes', 'container/bastion',
+            'selenium/firefox', 'selenium/chrome', 'verify/rhel-7-7',
+        ],
+        'rhel-8.0': ['avocado/fedora', 'container/bastion', 'selenium/firefox', 'selenium/chrome',
+            'verify/rhel-8-0',
+        ],
+        'rhel-8-appstream': ['avocado/fedora', 'container/bastion', 'selenium/firefox',
+            'selenium/chrome', 'verify/rhel-8-0', 'verify/rhel-8-1',
+        ],
+        'rhel-8.1': ['verify/rhel-8-1',
+        ],
+        '_manual': ['verify/continuous-atomic', 'verify/fedora-i386', 'verify/fedora-testing',
+        ],
+    },
+    'cockpit-project/starter-kit': {
+        'master': [
+            'cockpit/centos-7',
+            'cockpit/fedora-29',
+        ],
+    },
+    'cockpit-project/cockpit-ostree': {
+        'master': [
+            'cockpit/fedora-atomic',
+            'cockpit/continuous-atomic',
+            'cockpit/rhel-atomic',
+        ],
+    },
+    'cockpit-project/cockpit-podman': {
+        'master': [
+            'cockpit/fedora-29',
+        ],
+    },
+    'weldr/welder-web': {
+        'master': [
+            'cockpit/fedora-29/chrome',
+            'cockpit/fedora-29/firefox',
+            'cockpit/rhel-7-7/firefox',
+            'cockpit/rhel-8-1/chrome',
+            'cockpit/fedora-30/edge',
+        ],
+    }
 }
 
-# Test context for projects that use Cockpit's non-public images
-REDHAT_EXTERNAL_PROJECTS = {
-    'cockpit-project/cockpit-ostree': [
-        'cockpit/rhel-atomic',
-    ],
-    'weldr/welder-web': [
-        'cockpit/rhel-7-7/firefox',
-        'cockpit/rhel-8-1/chrome',
-        'cockpit/fedora-30/edge',
-    ],
-}
+INTERNAL_CONTEXTS = [
+    'verify/rhel-7-6',
+    'verify/rhel-7-6-distropkg',
+    'verify/rhel-7-7',
+    'verify/rhel-8-0',
+    'verify/rhel-8-1',
+    'verify/rhel-atomic',
+    'selenium/edge',
+    'cockpit/rhel-7-7/firefox',
+    'cockpit/rhel-8-1/chrome',
+    'cockpit/fedora-30/edge',
+    'cockpit/rhel-atomic',
+]
```

Comment:
I don't quite like the duplication here. I think a helper function like `is_redhat_context()` that is true if the context contains the word "rhel" or "edge" will need less maintenance. But this is fine for a follow-up if you prefer.

Reply:
I can change it easily, no problem. I was just not sure if such checking would be sufficient.

Reply:
Yeah, IMHO it's easier to define the rule which images need to be private than having to spell all of them out (this is prone to errors, and we did errors there several times).